_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -314,6 +314,14 @@ typedef struct tagENCODE_CAPS
     UCHAR    MaxNum_WeightedPredL1;
 } ENCODE_CAPS;
 
+// We cannot modify ENCODE_CAPS structure, but need to add new features
+struct MFX_ENCODE_CAPS
+{
+    ENCODE_CAPS ddi_caps;
+    bool CQPBRCSupport: 1;
+    bool CBRSupport   : 1;
+    bool VBRSupport   : 1;
```

Comment:
no, please don't do it. do not introduce bitfields without strict requirement. Convert to simple bools:
```
    bool CQPBRCSupport;
    bool CBRSupport;
    bool VBRSupport;
```

Reply:
Sorry, mistake after refactored. Done.